utils/config-utils.js

```diff
@@ -0,0 +1,10 @@
+// `CUTTLE_ENV` is needed to test prod specific files without actually putting the app in to
+// a production state, which requires an actual database connection
+const isProd =
+  process.env.CUTTLE_ENV !== undefined
+    ? process.env.CUTTLE_ENV === 'production'
+    : process.env.NODE_ENV === 'production';
```

Comment:
This is a new utility I added for checking for a prod environment, it will prefer the `CUTTLE_ENV` if set, otherwise if it is not set it will default to whatever value is in the NODE_ENV. Once we figure out all of the `NODE_ENV` stuff we can probably get rid of this abstraction.